highlightjs-glimmer/package.json

```
@@ -26,7 +26,7 @@
     "lint:prettier:fix": "prettier --write ."
   },
   "engines": {
-    "node": "^14 || ^16 || ^18"
+    "node": "^14 || ^16 || ^18 || ^20"
```

```suggestion
    "node": "^14 || ^16 || ^18 || >= 20"
```

this pattern is pretty common, `>=` on the most recent lts, so that folks can play with 21 or whatever without issue

---

the main thing is that I don't want to support 15, 17, etc